test/runtime/Policies.go

```diff
@@ -1155,6 +1151,14 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 			// is still to be completed (GH-3393).
 			By(fmt.Sprintf("Flushing global connection tracking table before importing policy"))
 			vm.FlushGlobalConntrackTable().ExpectSuccess("Unable to flush global conntrack table")
+		})
+
+		AfterEach(func() {
+			vm.PolicyDelAll().ExpectSuccess("Clearing policy after egress test")
```

Comment:
Nit: Can we make this `Failed to clear policy after egress test` to make it more obvious in logs that something failed.